src/scss/_mixins.scss

```diff
@@ -129,9 +129,13 @@
   width: $marker-dimension;
 }
 
-@mixin focusable {
+@mixin focusable($inset: false) {
   &:focus {
-    box-shadow: $focus-element-box-shadow;
+    @if $inset == false {
+      box-shadow: $focus-element-box-shadow;
+    } @else {
+      box-shadow: inset $focus-element-box-shadow;
```

Comment:
The box shadow has to be inset for settings panel items as otherwise it is not visible (they are at the edge of the settings panel and it has the overflow hidden)